woocommerce/woocommerce
# Add new E2E tests to cover Shopper Cart Block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes milestone #[29](https://github.com/woocommerce/woocommerce-quality/milestone/29) in Woo Quality repo

Added new e2e tests to cover shopper cart block.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch locally
2. Execute them all one by one: 
`pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/cart-block.spec.js`
`pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/cart-block-calculating-shipping.spec.js`
`pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/cart-block-calculating-tax.spec.js`
`pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/cart-block-coupons.spec.js`
3. Make sure they are passing and optionally run the whole suite



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
